JavaScript/JavaScriptSDK/SendBuffer.ts

```diff
@@ -179,15 +180,23 @@ module Microsoft.ApplicationInsights {
         }
 
         private removePayloadsFromBuffer(payloads: string[], buffer: string[]): string[] {
-            var cleared: string[] = [];
+            var remaining: string[] = [];
```

Comment:
just wanted to make sure you didn't forget unit tests for this change (or it's covered by existing ones)